### ISSUE TITLE
admin panel not changing items

### ISSUE THREAD

**Issue by huss** [OpenEnergyDashboard/OED#261]
I am having two issues on the development server version:
1) When I first change the display name it looks fine. If you refresh the page or open in another browser then there is no name at all.
2) When I change the default graph type and hit submit it does not seem to have any impact.

**Comment by johndiiorio** [comment]
Regarding 1, did you select "submit" after changing the display name? Submitting is required for all preferences.

Regarding 2, does a refresh on the home page correctly show the chosen chart type? This is by design - the administrator only changes the default chart type, not the current chart type.

**Comment by huss** [comment]
>
> Regarding 1, did you select "submit" after changing the display name?
> Submitting is required for all preferences.
>
​Yes I did the submit. Right after that the name did change but not after a
refresh (goes blank). From what I see there is no longer any name showing
on the dashboard on the development server. I tried another web browser and
I don't see a name there either.​


> Regarding 2, does a refresh on the home page correctly show the chosen
> chart type? This is by design - the administrator only changes the default
> chart type, not the current chart type.
>
I'm not sure what is up. When I submit then that button is no longer active
unless I choose a new option. However, it does not seem to stick in that
web browser. I tried another web browser and I did not see it change from a
line graph. I don't see any cookies so I assume this was a good test but am
not certain.​ Maybe I'm doing something wrong?
